go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/IoTOpen/go-lynx
 go 1.19
 
 require (
-	github.com/eclipse/paho.mqtt.golang v1.3.3
+	github.com/eclipse/paho.mqtt.golang v1.4.3
 	github.com/spf13/viper v1.8.1
```

Comment:
There should be updates for viper as well

Reply:
1.8.1 => 1.16.0